api/nodes/views.py

```diff
@@ -704,11 +738,209 @@ def perform_destroy(self, instance):
+        registration_metadata      dictionary         dictionary of question ids and responses from registration schema
+        datetime_initiated         iso8601 timestamp  timestamp that the draft was created
+        datetime_updated           iso8601 timestamp  timestamp when the draft was last updated
+
+    ##Relationships
+
+    ###Branched From
+
+    Node that the draft is branched from.  The node endpoint is available in `/branched_from/links/related/href`.
+
+    ###Initiator
+
+    User who initiated the draft registration.  The user endpoint is available in `/initiator/links/related/href`.
+
+    ##Registration Schema
+
+    Detailed registration schema.  The schema endpoint is available in `/registration_schema/links/related/href`.
+
+    ##Actions
+
+    ###Create Draft Registration
+
+        Method:        POST
+        URL:           /links/self
+        Query Params:  <none>
+        Body (JSON):   {
+                        "data": {
+                            "type": "draft_registrations",  # required
+                            "attributes": {
+                                "registration_supplement": {schema_name}, # required
+                                "registration_metadata": {"question_id": {"value": "question response"}} # optional
+                            }
+                        }
+                    }
+        Success:       201 OK + draft representation
+
+    To create a draft registration, issue a POST request to the `self` link.  Registration supplement must be the name of an
+    active registration schema, for example, "Open-Ended Registration".  Registration metadata is not required on the creation
+    of the draft. If registration metadata is included, it must be a dictionary with keys as question ids in the registration
+    supplement, and values as nested dictionaries matching the specific format in the registration schema.  See registration schema
+    endpoints for specifics. If question is multiple-choice, question response must exactly match one of the possible choices.
+
+    ##Links
+
+    See the [JSON-API spec regarding pagination](http://jsonapi.org/format/1.0/#fetching-pagination).
+
+    #This request/response
+
+    """
+    permission_classes = (
+        IsAdmin,
+        drf_permissions.IsAuthenticatedOrReadOnly,
+        base_permissions.TokenHasScope,
+        ExcludeRetractions
+    )
+
+    required_read_scopes = [CoreScopes.NODE_DRAFT_REGISTRATIONS_READ]
+    required_write_scopes = [CoreScopes.NODE_DRAFT_REGISTRATIONS_WRITE]
+
+    serializer_class = DraftRegistrationSerializer
+    view_category = 'nodes'
+    view_name = 'node-draft-registrations'
+
+    # overrides ListCreateAPIView
+    def get_queryset(self):
+        node = self.get_node()
+        drafts = DraftRegistration.find(Q('branched_from', 'eq', node))
+        return [draft for draft in drafts if not draft.registered_node or draft.registered_node.is_deleted]
```

Comment:
Could move the `not draft.registered_node` into the modm query:
`Q('registered_node', 'eq', None)`